tokens/contracts/access/MinterAccessControl.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/EnumerableSetUpgradeable.sol";
+
+abstract contract MinterAccessControl is Initializable, OwnableUpgradeable {
+    using EnumerableSetUpgradeable for EnumerableSetUpgradeable.AddressSet;
+
+    EnumerableSetUpgradeable.AddressSet private _minters;
+    
+    event MinterGranted(address indexed account);
+    event MinterRevoked(address indexed account);
+
+    function __MinterAccessControl_init() internal initializer {
+        __Ownable_init_unchained();
+        __MinterAccessControl_init_unchained();
+    }
+
+    function __MinterAccessControl_init_unchained() internal initializer {
+    }
+
+    /**
+     * @dev Add `_minter` to the list of allowed minters.
+     */
+    function grantMinter(address _minter) external onlyOwner {
```

Comment:
I think is better to change "**addMinter**" for method name **grantMinter**

Reply:
done